src/app/shared/model/feeder-podcast.model.ts

```diff
@@ -54,7 +55,12 @@ export class FeederPodcastModel extends BaseModel {
     }
     this.link = this.doc['link'] || '';
     this.newFeedUrl = this.doc['newFeedUrl'] || '';
-
+    if (this.series.has('prx:account')) {
+      this.series.follow('prx:account').subscribe(account => this.authorName = account['name']);
+    }
+    if (this.doc['author']) {
+      this.authorName = this.doc['author']['name'];
```

Comment:
It could be something weird about my account, but for me doc has an author but the author Object has no properties, so this.authorName is being set to undefined. (It was previously === "Sandi Barr" from the account['name'] though.)